recipes/validators/meta.yaml

```diff
@@ -3,19 +3,19 @@
 # Jinja variables help maintain the recipe as you'll update the version only here.
 # Using the name variable with the URL in line 14 is convenient
 # when copying and pasting from another recipe, but not really needed.
-{% set name = "simplejson" %}
```

Comment:
Please don't move the `example/meta.yaml` recipe, but just copy it.
Best thing would be to amend commit f49b4603356b619689212ff76dcf45ca04254436 such that `example/meta.yaml` is restored and then force push that change. That way we won't have the move in the Git history (to avoid confusions).